### DISCUSSION TITLE
New format for cpmg_ch3_mq?

### DISCUSSION THREAD

**Discussion by FitzkeeLab** [gbouvignies/ChemEx#72] [Q&A] [answered]
Hi! 

I'm trying to run the example "cpmg_ch3_mq" file from the website - nothing special, just trying to run the dataset that's already there. However, when I do so, I get:

```
Reading Experimental Data
=========================

File Name                                     Experiment                Profiles                 
---------                                     ----------                --------                 
Experiments/600mhz.toml                       
In the section 'experiment' of Experiments/600mhz.toml, ''type'' must be provided!
```

I noticed that one of the recent announcements mentioned that some of the file formats have changed, and I suspect that's what's going on here. But the documentation for this experiment (https://gbouvignies.github.io/ChemEx/docs/experiments/cpmg/cpmg_ch3_mq) does not include anything like "type." Am I missing something?

I'm open to using an older version of the software it it would be easier while the new documentation is written. I'm trying to get some preliminary data for a proposal, and fitting is the final hurdle.

Thanks!
Nick

**Comment by FitzkeeLab** [comment, marked as answer]
Hah! Got it. Turns out when I used pip it grabbed the older version of chemex, so the problem was that my chemex was old (2018) not that my script was old. Seems to be working now...

Nick

**Comment by gbouvignies** [comment]
Hi Nick,

Just wondering, did you get chemex 2108 by default after `pip install chemex`? If so, what version of Python was on your machine? Which OS were you running?

Best,
Guillaume

**Reply by FitzkeeLab** [reply]
Hi Guillaume,

My Python was version 3.8, so I think getting the 2018 version was entirely expected. I’m running Ubuntu 20.04 LTS, and the system python hasn’t caught up to 3.9 yet.

Ultimately, I used the python 3.9 distribution on Anaconda, and it worked just fine. My dataset’s pretty noisy, but I think I’m getting reasonable fits, and the software is working as expected.

That said, this is a new world for me – it’s one thing to read papers with CPMG data in them, but it’s quite another to be on the front lines. If you have any articles to suggest where they really get into the practical details of fitting, I’d appreciate it if you’d pass them along. One question I’m facing, in particular, is whether to include or exclude a residue from a global fit.

Thanks for following up!
Nick
